source/uwp/AdaptiveCardsNuget/AdaptiveCards.Renderer.Uwp.nuspec

```diff
@@ -1,16 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
-        <id>AdaptiveCards.Renderer.Uwp</id>
-        <version>1.0.0-beta1</version>
-        <authors>Microsoft</authors>
-        <owners>microsoft</owners>
```

Comment:
><owners>microsoft</owners> [](start = 8, length = 26)

Issue #897 mentions Microsoft should be a coowner. Is that something that should go somewhere in the nuspec?